src/error/mod.rs

```diff
@@ -367,6 +368,14 @@ impl From<InvalidNameError> for FederationError {
     }
 }
 
+impl From<FederationSpecError> for FederationError {
+    fn from(_err: FederationSpecError) -> Self {
+        // TODO: When we get around to finishing the composition port, we should really switch it to
+        // using FederationError instead of FederationSpecError.
+        todo!()
```

Comment:
Yeah, we never actually tested it so I just left it `todo!()` since the entire thing will probably get removed anyway once we consolidate error types in the future.